tests/libpanda/safety_helpers.h

```diff
@@ -168,6 +168,14 @@ void set_honda_fwd_brake(bool c){
   honda_fwd_brake = c;
 }
 
+void set_stock_aeb(bool c){
+  stock_aeb = c;
+}
+
+bool get_stock_aeb(){
+  return stock_aeb;
+}
+
```

Comment:
We might be able to combine honda's fwd_brake with a generic variable

Reply:
Will make new PR for AEB stuff